examples/c/chain_model/chain_model.h

```diff
@@ -22,23 +22,23 @@
 extern "C" {
 #endif
 
-#define X0_NM2_FILE "/home/gianluca/acados/examples/c/chain_model/x0_nm2.txt"
-#define X0_NM3_FILE "/home/gianluca/acados/examples/c/chain_model/x0_nm3.txt"
-#define X0_NM4_FILE "/home/gianluca/acados/examples/c/chain_model/x0_nm4.txt"
-#define X0_NM5_FILE "/home/gianluca/acados/examples/c/chain_model/x0_nm5.txt"
-#define X0_NM6_FILE "/home/gianluca/acados/examples/c/chain_model/x0_nm6.txt"
-#define X0_NM7_FILE "/home/gianluca/acados/examples/c/chain_model/x0_nm7.txt"
-#define X0_NM8_FILE "/home/gianluca/acados/examples/c/chain_model/x0_nm8.txt"
-#define X0_NM9_FILE "/home/gianluca/acados/examples/c/chain_model/x0_nm9.txt"
-
-#define XN_NM2_FILE "/home/gianluca/acados/examples/c/chain_model/xN_nm2.txt"
-#define XN_NM3_FILE "/home/gianluca/acados/examples/c/chain_model/xN_nm3.txt"
-#define XN_NM4_FILE "/home/gianluca/acados/examples/c/chain_model/xN_nm4.txt"
-#define XN_NM5_FILE "/home/gianluca/acados/examples/c/chain_model/xN_nm5.txt"
-#define XN_NM6_FILE "/home/gianluca/acados/examples/c/chain_model/xN_nm6.txt"
-#define XN_NM7_FILE "/home/gianluca/acados/examples/c/chain_model/xN_nm7.txt"
-#define XN_NM8_FILE "/home/gianluca/acados/examples/c/chain_model/xN_nm8.txt"
-#define XN_NM9_FILE "/home/gianluca/acados/examples/c/chain_model/xN_nm9.txt"
+#define X0_NM2_FILE "/home/bstickan/temp/acados/examples/c/chain_model/x0_nm2.txt"
```

Comment:
This file is templated by CMake,
this file shold not be tracked by git.

Not sure if it is templated by Makefile,
maybe it should be added there.

Reply:
It is only templated by cmake. And actually now it is not even used at all, since I hand coded the initial values in a `.c` file instead. But I would keep this file as a reference for where the data in the `.c` file comes from (i.e. the `.m` script generated the `.txt` data files). If you feel like directly generating the `.c` files from the `.m` script, then afterwards you can remove this header completely.